notebooks/source/index.rst

```diff
@@ -0,0 +1,23 @@
+.. Pyro Tutorials documentation master file, created by
+   sphinx-quickstart on Tue Oct 31 11:33:17 2017.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to NumPyro Examples and Tutorials!
+==========================================
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Examples:
+
+   bayesian_regression
+   logistic_regression
```

Comment:
Can we remove logistic_regression for now? This was mostly used for internal benchmarking and is not likely to be useful as a reference, unless we add more description to it and carefully compare performance across implementations in other frameworks (right now we are just comparing with reported numbers in the paper).